server/crud/__init__.py

```diff
@@ -1,2 +1,2 @@
-from .crud_buoy import buoy  # noqa
-from .crud_coastline import coastline  # noqa
+from server.crud.crud_buoy import buoy  # noqa: F401
+from server.crud.crud_coastline import coastline  # noqa: F401
```

Comment:
why add the relative path instead of having the absolute?

Reply:
See my response [here](https://github.com/clairBuoyant/server/pull/40#issuecomment-1203529181).